apps/studio/data/database-extensions/database-extension-disable-mutation.ts

```diff
@@ -0,0 +1,70 @@
+import { useMutation, UseMutationOptions, useQueryClient } from '@tanstack/react-query'
+import { toast } from 'react-hot-toast'
+
+import { del } from 'data/fetchers'
+import { ResponseError } from 'types'
+import { databaseExtensionsKeys } from './keys'
+
+export type DatabaseExtensionDisableVariables = {
+  projectRef: string
+  connectionString: string
+  id: string
+}
+
+export async function disableDatabaseExtension({
+  projectRef,
+  connectionString,
+  id,
+}: DatabaseExtensionDisableVariables) {
+  let headers = new Headers()
+  if (connectionString) headers.set('x-connection-encrypted', connectionString)
+
+  const { data, error } = await del('/platform/pg-meta/{ref}/extensions', {
+    params: {
+      header: { 'x-connection-encrypted': connectionString },
+      path: { ref: projectRef },
+      query: { id },
+    },
+    headers: Object.fromEntries(headers),
```

Comment:
Same comment as in `apps/studio/data/database-extensions/database-extensions-query.ts`